CHANGES.md

```diff
@@ -14,8 +14,10 @@
 
 ### Changes
 
+* remove `pyaml` python dependency.
```

Comment:
We should probably say, remove `pyaml` and directly use `pyyaml` instead so people know what happened.